lib/jekyll/polyglot/patches/jekyll/site.rb

```diff
@@ -125,14 +132,26 @@ def relative_url_regex
       @exclude.each do |x|
         regex += "(?!#{x}\/)"
       end
-      url_quoted = config['url']
-      url_quoted = Regexp.quote(url_quoted) unless url_quoted.nil?
-      %r{href=\"(?:#{url_quoted})?#{@baseurl}\/((?:#{regex}[^,'\"\s\/?\.#]+\.?)*(?:\/[^\]\[\)\(\"\'\s]*)?)\"}
+      %r{href=\"?#{@baseurl}\/((?:#{regex}[^,'\"\s\/?\.#]+\.?)*(?:\/[^\]\[\)\(\"\'\s]*)?)\"}
```

Comment:
As far as I can tell, relative URLs like `/en/about` will still be processed, while absolute URLs that include `/:lang` are excluded. Is that intentional?
Should this be documented/tested?